package.json

```
@@ -67,7 +67,7 @@
     "testdouble": "^3.16.1",
     "ts-jest": "^26.2.0",
     "ts-loader": "^8.0.2",
-    "typescript": "~3.9.7"
+    "typescript": "^3.9.7"
   },
   "peerDependencies": {
     "cypress": "^4.12.1",
```

This will need to be updated too
```suggestion
    "cypress": "^5.0.0",
```